### ISSUE TITLE
Day labor next steps

### ISSUE THREAD

**Issue by desigonz** [cityofaustin/techstack#2824]
- [x] get form pushed to prod
- [x] Once that's done, @Jo-Dwyer to add button on service page 
- [x] Then @desigonz to send intro email to Jose/Derrick and loop in Jo, for their final review/approval
- [x] When we publish, also get devs to add the right email addresses to the form

**Comment by desigonz** [comment]
@Jo-Dwyer We have links for the Day Labor form! Can you add buttons to English and Spanish versions?

https://forms.austin.gov/request-day-laborer/form
https://forms.austin.gov/solicite-un-jornalero/form

**Comment by Jo-Dwyer** [comment]
@easherma As I understand it, buttons should be related to highlighted text, not to the whole text block. 

I selected the desired button text and used the Button feature... 
    ![Screen Shot 2019-09-09 at 11.51.45 AM.png](https://images.zenhubusercontent.com/5d641145b9bd2f17c566338c/0911ccee-a291-47da-86de-dda2076e70f0)
...and then previewed it:
    ![Screen Shot 2019-09-09 at 11.50.16 AM.png](https://images.zenhubusercontent.com/5d641145b9bd2f17c566338c/e02c4c79-d71c-4296-91fe-c8d1a0415012)

Problem is likely user error: please assist.

**Comment by Jo-Dwyer** [comment]
If someone fixed the button situation, great. If not, I seem to have gotten out of my own way. Buttons are added. I used "empezar" for the start button on SP version.  

https://alpha.austin.gov/en/preview/services/UGFnZVJldmlzaW9uTm9kZToxNTI3

**Comment by Jo-Dwyer** [comment]
@srigdon @ablangworthy27 @annieszot can one of you take a look at the Joplin draft, both English and Spanish?

**Comment by ablangworthy27** [comment]
I can look at it and review for English. Then, it will have to be passed off to Desi for Spanish.

**Comment by desigonz** [comment]
Happy to review the Spanish! Also, Jo, I’ll find you after my meeting to talk about that button button.

--

Desi Gonzalez
Product Manager
Office of Design and Delivery
desi.gonzalez@austintexas.gov
(512) 766-7377
http://odd.austintexas.io/

**Comment by Jo-Dwyer** [comment]
@desigonz it seems as though it was user error since it nows seems like it is formatted correctly. (user error likely.)

**Comment by Jo-Dwyer** [comment]
Thanks for taking on the first edit, @ablangworthy27 !

**Comment by ablangworthy27** [comment]
@Jo-Dwyer, my only two notes on the English are the following:

1. format of the times to match the style guide. I think you are already fixing this as I type.
2. This sentence struck me as one that could be fine tuned.

> 
"Additionally, if your job site is further than 10 miles from the Day Labor Center, we recommend paying an additional $10 to go towards gas money for your worker."

Finding a way to remove one of those "additionals" would fix it, I think. This is my suggestion:

"If your job site is further than 10 miles from the Day Labor Center, we also recommend paying your worker an additional $10 for gas."

**Comment by desigonz** [comment]
Oooh something I'm noticing @Jo-Dwyer: this is a service page with only one step, but we've talked about how service pages should have at least two steps. (This isn't required in Joplin right now but is in the dev backlog of things to implement.) 

Maybe something to review during content pin-up tomorrow? Want to make sure we're all on the same page about this!

**Comment by Jo-Dwyer** [comment]
@desigonz Ahhhh Ok. I see that OPO has two steps, the second of which is "learn more". Is this something we want to hold off on publishing til we work it out at pinup?

**Comment by Jo-Dwyer** [comment]
Still waiting on Spanish translation. English is in preview, ready to go.

**Comment by Jo-Dwyer** [comment]
Hire a day laborer form needs
- [ ] @Jo-Dwyer adds a description in English
- [ ] @Jo-Dwyer decide: does this live independently in the topic or only link from service page
- [ ] @desigonz translates to Spanish
- [ ] @desigonz connect Jo with Day Labor

**Comment by desigonz** [comment]
@Jo-Dwyer I made a placeholder translation! I'm planning to send some location content type stuff out for translation soon, so I can loop that in there.

I also went ahead and published the form page, since it's only affiliated with a department and therefore won't be linked to from anywhere. That meant that I was able to save the button links on the "hire a day laborer" service page. Here are the preview links:
- [Hire a day laborer—English](https://alpha.austin.gov/en/preview/services/UGFnZVJldmlzaW9uTm9kZToyMjMz)
- [Hire a day laborer—Spanish](https://alpha.austin.gov/es/preview/services/UGFnZVJldmlzaW9uTm9kZToyMjMz)

I'll draft an email to connect you to Day Labor and get the approvals going!
